production.Dockerfile

```diff
@@ -23,7 +23,7 @@
 #
 FROM node:18.12.1-bullseye-slim AS frontend-build
 WORKDIR /code
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+SHELL ["/bin/bash", "-e", "-o", "pipefail", "-c"]
```

Comment:
is this belt & braces or ??

Reply:
Was what I originally thought was the issue but it isn't. Nonetheless it doesn't seem to hurt